Source/ACE/Entity/SpellEntity.cs

```diff
@@ -0,0 +1,58 @@
+﻿using ACE.Entity;
+using ACE.Entity.Enum;
+using ACE.Factories;
+using ACE.Managers;
+using ACE.Network;
+using ACE.Network.Enum;
+using ACE.Network.GameMessages.Messages;
+using log4net;
+
+namespace ACE.Entity
+{
+    /// <summary>
+    /// Spell Entity
+    /// </summary>
+    public class SpellEntity :  MutableWorldObject
```

Comment:
This needs a better name.  Is there a particular reason a spell gets an entire class definition, though?  The only difference I see here is that it has a PlayScript implementation that creates an Effect.  There's nothing about the class definition that tells me it's a spell.  I think this could have been done in MutableWorldObject as is, with all the Physics values being passed into a constructor, and then checking the PhysicsData.PhysicsState in the PlayScript method.

Also, I recall seeing a video of multiple effects.  Was that not yours?  Is that implemented here?